scraper.rb

```diff
@@ -29,16 +34,23 @@ def scrape_page(page)
   end
 end
 
-# Load summary page.
-url = "https://epathway.monash.vic.gov.au/ePathway/Production/Web/GeneralEnquiry/EnquiryLists.aspx?ModuleCode=LAP"
-page = agent.get(url)
+base_url = "https://epathway.monash.vic.gov.au/ePathway/Production/Web/GeneralEnquiry/"
+entry_form_url_ext = "/EnquiryLists.aspx?ModuleCode=LAP"
+first_page_url = base_url + entry_form_url_ext
+
+# Get first page with radio buttons.
+page = agent.get(first_page_url)
+form = page.forms.first
+form.radiobuttons[0].check
+page = form.submit(form.button_with(:value => "Next"))
 
 # Now do the paging magic
 number_pages =  page.at("#ctl00_MainBodyContent_mPagingControl_pageNumberLabel").inner_text.split(" ")[3].to_i
 
 (1..number_pages).each do |no|
-  url = "https://epathway.monash.vic.gov.au/ePathway/Production/Web/GeneralEnquiry/EnquirySummaryView.aspx?PageNumber=#{no}"
-  page = agent.get(url)
+  result_page_extension = "/EnquirySummaryView.aspx?PageNumber=#{no}"
+  results_page_url = base_url + result_page_extension
+  page = agent.get(results_page_url)
   puts "Scraping page #{no} of " + number_pages.to_s + "..."
```

Comment:
Doesn't feel like this would be very helpful debug output to me. I recon printing the record that it's currently trying to save would be more useful—but if you were debugging it you could always add that I guess.